spacy/ml/models/spancat.py

```diff
@@ -17,6 +19,14 @@ def build_linear_logistic(nO=None, nI=None) -> Model[Floats2d, Floats2d]:
     return chain(Linear(nO=nO, nI=nI, init_W=glorot_uniform_init), Logistic())
 
 
+@registry.layers("spacy.Softmax.v1")
```

Comment:
Can't you use the `Softmax.v1` layer from thinc without needing an extra definition here?